Neos.Flow/Classes/Mvc/Controller/ActionController.php

```diff
@@ -462,7 +462,7 @@ protected function callActionMethod()
         $validationResult = $this->arguments->getValidationResults();
 
         if (!$validationResult->hasErrors()) {
-            $actionResult = call_user_func_array([$this, $this->actionMethodName], $preparedArguments);
+            $actionResult = $this->{$this->actionMethodName}(...$preparedArguments);
```

Comment:
These cases are debatable... the `->{$this->*MethodName}()` syntax is not super nice. I still slightly prefer it over the `call_user_func_array`, but I'm not gonna fight for it.

Reply:
Mmm, IDK, we could do something like: 

```
$actionMethodName = $this->actionMethodName;
$this->$actionMethodName(...$preparedArguments);
```

But I don't think that's much easier readable... I think it's fine, it's odd anywys and might need a revisit at some point.